### PR TITLE
fix: map.toString is replaced with readableMap2string

### PR DESCRIPTION
**Pull request by OrrinHatch**

### CODE REVIEW

**Comment by OrrinHatch** [comment]
还有一些问题存在,先别合并

File: android/src/main/java/com/openimsdkrn/OpenImSdkRnModule.java [hunk 1]
```diff
@@ -423,9 +423,9 @@ public void createImageMessageFromFullPath(String imagePath, String operationID,
 
   @ReactMethod
   public void createImageMessageByURL(ReadableMap options, String operationID, Promise promise) {
-    String sourcePicture = Objects.requireNonNull(options.getMap("sourcePicture")).toString();
-    String bigPicture = Objects.requireNonNull(options.getMap("bigPicture")).toString();
-    String snapshotPicture = Objects.requireNonNull(options.getMap("snapshotPicture")).toString();
+    String sourcePicture = readableMap2string(Objects.requireNonNull(options.getMap("sourcePicture")));
+    String bigPicture = readableMap2string(Objects.requireNonNull(options.getMap("bigPicture")));
+    String snapshotPicture =readableMap2string(Objects.requireNonNull(options.getMap("snapshotPicture")));
```
Comment (lgz5689) [inline android/src/main/java/com/openimsdkrn/OpenImSdkRnModule.java:428]: Note formatting

Comment (lgz5689) [inline android/src/main/java/com/openimsdkrn/OpenImSdkRnModule.java:428]: Note formatting

**Comment by Bloomingg** [comment]
There are many other methods that also use map.toString() and you need to determine if you want to replace them all with readableMap2string

**Comment by lgz5689** [comment]
see https://github.com/openimsdk/open-im-sdk-reactnative/issues/32

**Pull request closed without merging**